newswires/client/src/SearchSummary.tsx

```diff
@@ -30,12 +30,20 @@ const presetLabel = (preset: string) => {
 	}
 };
 
+type SearchTermBadgeLabel =
```

Comment:
added to help linting to know whether we're checking all possible cases in the `switch` statement in `handleBadgeClick`